R/infix_spaces_linter.R

```diff
@@ -57,6 +58,13 @@ infix_metadata$low_precedence <- infix_metadata$string_value %in% c(
 # comparators come up in several lints
 infix_metadata$comparator <- infix_metadata$string_value %in% c("<", "<=", ">", ">=", "==", "!=")
 
+# undesirable_operator_linter needs to distinguish
+infix_overload <- data.frame(
+  exact_string_value = c("<-", "<<-", "=", "->", "->>", "^", "**"),
+  xml_tag = rep(c("LEFT_ASSIGN", "RIGHT_ASSIGN", "OP-CARET"), c(3L, 2L, 2L)),
```

Comment:
I remember `EQ_ASSIGN` to be a thing. Is that a thing for older R versions?

Reply:
already handled actually IINM, but added tests anyway. Non-assignment `=` were indeed wrongly handled.